meta-mbl-distro/conf/distro/mbl-production.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+##################################################################################
+# mbl-production.conf
+#    Based on the main mbl.conf distribution
+#    Sets some defaults production variables options via mbl-distro-production.inc
+##################################################################################
+
+require conf/distro/mbl.conf
```

Comment:
Should we introduce an "mbl-development" distro too?

And then have a "common" .inc file that  both the dev and prod distro confs `require`?

Reply:
My concern is that people will change mbl.conf because they want to change the dev distro and end up changing the production distro without realizing... And it's harder to test the production distro so it's harder to find out that something has accidentally changed.